components/sheet/TripFormSheet.tsx

```
@@ -0,0 +1,464 @@
+  const [name, setName] = useState('');
+  const [icon, setIcon] = useState('✈️');
+  const [currencyCode, setCurrencyCode] = useState<string | null>(null);
+  const [startDate, setStartDate] = useState(new Date());
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  const [hasEndDate, setHasEndDate] = useState(false);
+  const [isSaving, setIsSaving] = useState(false);
+
+  useEffect(() => {
+    if (trip) {
+      setName(trip.name);
+      setIcon(trip.icon);
```

Biome (2.1.2)

[error] This hook does not specify its dependency on resetForm.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)